core/tee/tee_svc.c

```diff
@@ -256,6 +281,371 @@ TEE_Result syscall_get_property(unsigned long prop, void *buf, size_t blen)
 	}
 }
 
+static TEE_Result get_prop_tee_dev_id(struct tee_ta_session *sess,
```

Comment:
I think you should use the new static functions in `syscall_get_property_obsolete()` when possible, to avoid code duplication.